src/azure-cli/azure/cli/command_modules/appservice/_params.py

```diff
@@ -170,8 +172,8 @@ def load_arguments(self, _):
         c.argument('slot', options_list=['--slot', '-s'], help='Name of the web app slot. Default to the productions slot if not specified.')
 
     with self.argument_context('webapp list-runtimes') as c:
```

Comment:
The functionapp equivalent has the same params as the webapp one, besides the deprecated `--linux` param (since the functionapp command was newly added)

<img width="1139" alt="Screen Shot 2022-02-08 at 2 18 26 PM" src="https://user-images.githubusercontent.com/10121214/153085104-bd6132ec-8dcb-4105-9ca2-827829688413.png">